xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
You were abundantly clear. I did my best to point you to documentation which would answer your question, which was (as your clarification again shows) based on a misunderstanding of Python descriptors. It is not a write-only file system. You interact with a file system by opening files and writing to them, not by assigning to the file system itself. It is not at all clear what assignment to a file system would mean, so that operation is unsupported.

I would suggest looking over the code of file thumbs or the Recommender, as you had previously indicated you might do. Both write to the file system. I would also look at pyfilesystem documentation.